package.json

```diff
@@ -27,6 +27,7 @@
         "express": "4.16",
         "lru-cache": "5.1.1",
         "mdn-browser-compat-data": "0.x",
+        "mdn-data": "2.x",
```

Comment:
Years of having used Node and stuff and I'm still learning! I had no idea you can use an `x` like that. Thanks! I love it.

By the way, how is that different from `~`? I.e. `"~2.0"` which would match, for example, "2.0.2", "2.0.3", "2.1.0", "2.9", etc.

Reply:
It’s that you can use `1.2.x` to specify `>=1.2 <1.3`, which means the same thing as `~1.2`:

https://docs.npmjs.com/misc/semver#x-ranges-12x-1x-12-

Also, `~2.0` means `>=2.0 <2.1`, so it would only match `2.0.2` and `2.0.3`, but not `2.1` or `2.9`.

To get what you mean you’d use `~2` which means the same thing as `2.x`.

Reply:
To my eyes `2.x` looks a more obvious/user-friendly than `~2`. 

To me, the ~ usually means home directory or "approximately". When say "~5 seconds" then "4.87 sec" matches that.